app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/address/single_mixed_use_summary.scala.html

```diff
@@ -36,7 +35,7 @@
 @cyaRow(
         messages("returns.company-address.cyaLabel.company-address"),
         addressDisplay(answers.address, DisplayFormat.Block),
-        Some(cyaChange(messages("returns.company-address.cyaChange.company-address"), routes.CompanyDetailsController.isUk().url)),
+        Some(cyaChange(messages("returns.company-address.cyaChange.company-address"), returns.address.routes.MixedUsePropertyDetailsController.enterPostcode().url)),
```

Comment:
shouldn't this be the cya endpoint? If it's complete, we want to see the cya page first.

Reply:
also `returns.company-address.cyaChange.company-address` doesn't seem like the correct message here - you may need a new one